sql/hive/benchmarks/OrcReadBenchmark-jdk11-results.txt

```diff
@@ -3,154 +3,220 @@ SQL Single Numeric Column Scan
 
 OpenJDK 64-Bit Server VM 11.0.13+8-LTS on Linux 5.11.0-1022-azure
-Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
+Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
 Int and String Scan:                      Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Native ORC MR                                      2501           2561          85          4.2         238.5       1.0X
-Native ORC Vectorized                              1350           1448         138          7.8         128.8       1.9X
-Hive built-in ORC                                  3032           3080          67          3.5         289.2       0.8X
+Native ORC MR                                      2009           2082         104          5.2         191.6       1.0X
+Native ORC Vectorized                              1300           1337          52          8.1         124.0       1.5X
+Hive built-in ORC                                  2423           2437          20          4.3         231.0       0.8X
 
 
 ================================================================================================
 Partitioned Table Scan
 ================================================================================================
 
 OpenJDK 64-Bit Server VM 11.0.13+8-LTS on Linux 5.11.0-1022-azure
-Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
+Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
 Partitioned Table:                        Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Data column - Native ORC MR                        1969           2024          76          8.0         125.2       1.0X
-Data column - Native ORC Vectorized                 448            520          57         35.1          28.5       4.4X
-Data column - Hive built-in ORC                    2335           2407         102          6.7         148.4       0.8X
-Partition column - Native ORC MR                    959            975          16         16.4          61.0       2.1X
-Partition column - Native ORC Vectorized             71            105          26        220.9           4.5      27.7X
-Partition column - Hive built-in ORC               1391           1415          33         11.3          88.5       1.4X
-Both columns - Native ORC MR                       1657           1735         111          9.5         105.3       1.2X
-Both columns - Native ORC Vectorized                330            434          87         47.7          21.0       6.0X
-Both columns - Hive built-in ORC                   1898           2079         257          8.3         120.7       1.0X
+Data column - Native ORC MR                        1068           1156         124         14.7          67.9       1.0X
```

Comment:
Although this is a known GitHub Action result limitation, this twice faster result always misleads me. :)